.github/workflows/slsa-builder-go.yml

```diff
@@ -7,16 +7,23 @@ env:
   GENERATED_BINARY: generated-go-binary
   #TODO: let caller upload ?
   REGISTRY: ghcr.io
+  BUILDER_URL: https://github.com/laurentsimon/slsa-on-github
```

Comment:
should this be changed after build-go is merged?

Reply:
oops, done.